lib/susedistribution.pm

```diff
@@ -264,7 +264,7 @@ sub init_consoles {
         $self->add_console('installation',   'tty-console', {tty => check_var('VIDEOMODE', 'text') ? 1 : 7});
         $self->add_console('install-shell2', 'tty-console', {tty => 9});
         # On SLE15 X is running on tty2 see bsc#1054782
-        $self->add_console('root-console', 'tty-console', {tty => sle_version_at_least('15') ? 6 : 2});
+        $self->add_console('root-console', 'tty-console', {tty => sle_version_at_least('15') && is_sle ? 6 : 2});
```

Comment:
I see the problem, in code like !check_var($version_variable, '12-SP4'). So for versions which are not sle at all it will return true always. I would also ass is_sle to sle_version_at_least, but I wonder if we should make any exception for staging, etc.?